package.json

```diff
@@ -79,7 +79,7 @@
     "ignore": [
       "build/**",
       "dist/**",
-      "examples/_js/**",
+      "examples/**/js/**",
```

Comment:
I don't think we don't want to ignore linting of our example JS files. Caught some lint errors in some of the components.